packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/nodeexpress/updateconfigurations.md

```diff
@@ -0,0 +1,44 @@
+Before you can start using the email authenticator, you need to enable it in your Okta org.
+
+### Add the email authenticator to your org
+
+1. Open the Admin Console for your org.
+1. Choose **Security > Authenticators** to show the available authenticators.
+1. Locate the **Email** authenticator in the list. The **Email** authenticator is added by default for new orgs, but if it doesn't exist, use **Add Authenticator** to add it.
+1. Find the **Email** authenticator in the list. If it doesn't exist, click on **Add Authenticator** to add it.
+1. For the **Email** Authenticator, click on the **Actions** menu and then click **Edit**.
+1. Select **Authentication and recovery** for **This authenticator can be used for**. Changing the value to **Authentication and recovery**  allows email for sign-ins and password recoveries.
+1. Click **Save** to save your changes.
+
+### Setup your application to use the email authenticator
+
+#### Add new authentication policy
+
+1. Open the Admin Console for your org.
+1. Choose **Security > Authentication Policies** to show the authentication policies for your org.
+1. Click on **Add a Policy**.
+1. In the **Add Authentication Policy** dialog window, enter a policy name (for example, "2FA Policy) and description.
```

Comment:
I think best practice would be to give the policy a name either matching the app's name as it is a test?

Reply:
For consistency, I'll defer to you on the actual implementation of these instructions.